### PR TITLE
Add Octopus CLI EAP to Worker Tools

### PR DESCRIPTION
**Pull request by benjimac93**
This commit adds the installation of Octopus CLI EAP version 1.5.1 to both Ubuntu 22.04 and Windows LTSC 2019 Dockerfiles, as well as updates the README files for both images to reflect this change. Additionally, tests were added to confirm that Octopus CLI EAP was installed correctly on both images.

I've referred to it throughout as "Octopus CLI EAP" as that's what is used on the Octopus Downloads page, unsure if it should be referred to as something else though.

### CODE REVIEW

File: ubuntu.22.04/README.md [hunk 1]
```diff
@@ -38,6 +38,7 @@
 - Java Jdk 11.0.17+8-1ubuntu2~22.04
 - Kubectl 1.18.8-00
 - Kubelogin v0.0.25
+- Octopus CLI EAP 1.4.0
```
Comment (hnrkndrssn) [inline ubuntu.22.04/README.md:41]:
```suggestion
- Octopus CLI EAP 1.5.0

```

File: windows.ltsc2019/README.md [hunk 2]
```diff
@@ -37,6 +37,7 @@
 - Kubectl 1.18.8
 - Kubelogin (azure-kubelogin) 0.0.25
 - Node 14.17.2
+- Octopus CLI EAP 1.4.0
```
Comment (hnrkndrssn) [inline windows.ltsc2019/README.md:40]:
```suggestion
- Octopus CLI EAP 1.5.0

```

**Review by johnsimons — APPROVED** [review]
Apart from @hnrkndrssn comment, everything else LGTM

**Comment by hnrkndrssn** [comment]
Hi @benjimac93, thank you for your contribution!

We had some issues with publishing the `1.5.0` version of the `cli`. We just published version `1.5.1`, could you update this PR to install this version instead please.

Cheers,
Henrik

**Comment by benjimac93** [comment]
> Hi @benjimac93, thank you for your contribution!
> 
> We had some issues with publishing the `1.5.0` version of the `cli`. We just published version `1.5.1`, could you update this PR to install this version instead please.
> 
> Cheers, Henrik

of course ! Done :)

**Comment by benjimac93** [comment]
Anything else needed from me to get this merged?

**Comment by hnrkndrssn** [comment]
> Anything else needed from me to get this merged?

Hey @benjimac93, we've been dealing with some CI issues for our WorkerTools that we're in the process of sorting out.

Nothing required from you at this time, will let you know if we need you to make any changes.

Cheers, and apologies for the delay in getting your contribution merged!

**Comment by benjimac93** [comment]
any news on this @hnrkndrssn ?

**Comment by hnrkndrssn** [comment]
Hey @benjimac93 my sincere apologies, this fell off my radar :| I'll have a look and get things moving on this today.

**Comment by hnrkndrssn** [comment]
@benjimac93 I've opened a PR to your repo that adds your change to our new Windows 2022 workertools image (as we've introduced that since you opened this PR) and also bumped the version that is installed to latest.

**Comment by benjimac93** [comment]
@hnrkndrssn looks good, I've bumped ubuntu22.04 to v1.7.1 too

**Comment by hnrkndrssn** [comment]
Just waiting for GitHub to process the new commits so we can get another build going.

**Comment by hnrkndrssn** [comment]
Hey @benjimac93 I'm not sure what is going on here but GitHub is not picking up the new commits and is just spinning the wheels. Would you mind closing this PR and opening a new PR and we can get your change merged in.

Apologies for the inconvenience and delay on this PR.

**Comment by benjimac93** [comment]
@hnrkndrssn sure can.

Weirdly GitHub now things I forked from a different repo and won't let me submit a fresh PR back into OctopusDeploy/WorkerTools 

Seems to have happened with a bunch of Forks I have not just for OD repos

image

I'll take a fresh fork and try from there

**Pull request closed without merging**